lib/rules/require-class-comment.js

```diff
@@ -316,9 +317,10 @@ module.exports = function(context) {
 				if (paramsToCheck.indexOf(p) === -1) {
 					context.report({
 						node: jsdocNode,
-						message: 'Non-matching JSDoc parameter \'{{name}}\'.',
+						message: 'Non-matching JSDoc parameter \'{{name}}\'. Found ones are \'{{paramsToCheck}}\'.',
```

Comment:
I think it can help finding the origin of a problem. Moreover, it helped me understand eslint rule implementation a bit